src/config.books.json

```diff
@@ -378,5 +378,8 @@
   },
   "6c2f7d1d-fbad-42e8-b977-9b903e6cb83b": {
     "defaultVersion": "c3c7128"
+  },
+  "823ae3e1-57c4-44c5-b54a-310091040cf6": {
+    "defaultVersion": "53c8ac6"
```

Comment:
Looks good to me too! It matches this which resolves with JSON (instead of the 404 earlier): https://openstax.org/apps/archive/20230828.164620/contents/823ae3e1-57c4-44c5-b54a-310091040cf6@53c8ac6.json